irr/src/os.cpp

```diff
@@ -160,7 +160,7 @@ void Printer::print(const c8 *message, ELOG_LEVEL ll)
 
 	// Android logcat restricts log-output and cuts the rest of the message away. But we want it all.
 	// On my device max-len is 1023 (+ 0 byte). Some websites claim a limit of 4096 so maybe different numbers on different devices.
-	const size_t maxLogLen = 1023;
+	const int maxLogLen = 1023;
 	size_t msgLen = strlen(message);
 	size_t start = 0;
 	while (msgLen - start > maxLogLen) {
```

Comment:
I bet this produces a signed comparison warning now, so you'd have to change the other types too

Reply:
It actually doesn't, not in my testing at least. I believe the warning might not be (and probably should not be?) enabled, at least not for Android builds.

Anyhow, I've made `maxLogLen` a `size_t` again and used a `static_cast<int>(maxLogLen)` instead so this is not an issue.